### PR TITLE
fix(api): add requestid to webhooks

### PR DESCRIPTION
**Pull request by Orta21**
Ref. metriport/metriport-internal#1669

Ticket: #1889 

### Dependencies

- Upstream: none
- Downstream: none

### Description

Add requestIid to webhooks

### Testing

- Local
  - [x] Send webhook see in logs
- Staging
  - [ ] See logs in posthog
- Production
  - [ ] Monitor logs

Check each PR.

### Release Plan

- [ ] Merge this

### CODE REVIEW

**Review by RamilGaripov — APPROVED** [review]
LGTM

File: packages/api/src/command/medical/patient/consolidated-get.ts [hunk 1]
```diff
@@ -67,8 +69,18 @@ export async function startConsolidatedQuery({
...
   const progress: QueryProgress = { status: "processing", startedAt };
 
+  analytics({
+    distinctId: patient.cxId,
+    event: EventTypes.consolidatedQuery,
+    properties: {
+      patientId: patient.id,
+      requestId,
+    },
+  });
```
Comment (leite08) [inline packages/api/src/command/medical/patient/consolidated-get.ts:82]: We're sending this one here but there's no code to send another one with the duration. I assume this is the case you mentioned that needed to do Posthog SQL to calculate it, right?

Couldn't we have this on our code (send duration and not have SQL) if check the result of `processRequest()` [here](https://github.com/metriport/metriport/pull/1930/files#diff-5d3732ab972684ed41efa4877291f021f772141baddff08a616b2db2eb0181d4L78) and decide whether consolidated query worked or not?

File: packages/api/src/command/medical/document/document-query.ts [hunk 2]
```diff
@@ -89,6 +90,15 @@ export async function queryDocumentsAcrossHIEs({
...
   });
 
+  analytics({
+    event: EventTypes.documentQuery,
+    distinctId: cxId,
+    properties: {
+      requestId,
+      patientId,
+    },
+  });
```
Comment (leite08) [inline packages/api/src/command/medical/document/document-query.ts:100]: What's this for? So we can calculate the total across all HIEs?

File: packages/api/src/command/medical/patient/consolidated-get.ts (hunk 1 above)
Comment (Orta21) [reply packages/api/src/command/medical/patient/consolidated-get.ts:82]: We have the duration one on line 168 and 182

File: packages/api/src/command/medical/document/document-query.ts (hunk 2 above)
Comment (Orta21) [reply packages/api/src/command/medical/document/document-query.ts:100]: The reason for this one and the one about you commented on that only have properties `requestId` and `patientId` is so we can calculate 
-  num DQs with success WH sent
-  num CQs with success WH sent

We dont have the requestId when calling the endpoint so i add this one as the point of reference

File: packages/api/src/command/medical/patient/consolidated-get.ts (hunk 1 above)
Comment (Orta21) [reply packages/api/src/command/medical/patient/consolidated-get.ts:82]: As for why it exists look at the thread below

**Review by leite08 — APPROVED** [review]

**Pull request merged**